rolling/distribution.yaml

```diff
@@ -4941,6 +4941,11 @@ repositories:
       url: https://github.com/ros/robot_state_publisher.git
       version: rolling
     status: maintained
+  robotont_msgs:
+    doc:
+      type: git
+      url: https://github.com/robotont/robotont_msgs.git
+      version: ros2-rolling-devel
```

Comment:
Can you please add this in a separate PR?  It is a different package and to a different distribution.

Reply:
Removed